src/App.js

```diff
@@ -2,13 +2,39 @@ import React from "react";
 import "./App.css";
 
 class App extends React.Component {
+  constructor() {
+    super()
+    this.state = {
+      score: 0,
+      clicky: 1
+    }
+  }
+  onClicking = () => {
+    this.setState({ score: this.state.score + this.state.clicky })
+  }
+  prompt = () => {
+    alert("You can't afford that!")
+  }
+  currentScore = () => {
+    this.state.clicky = this.state.clicky + 1
+    this.setState({ score: this.state.score - 10 })
+  }
+  replay = () => {
+    this.state.clicky = 1
+    this.setState({ score: 0 })
+  }
```

Comment:
```suggestion
  replay = () => {
    this.setState({ score: 0, clicky: 1 })
  }
```
line 23 is modifying state directly. Modifying state directly can lead to weird bugs down the road and makes for extremely unreliable behavior in a React app. It's bad practice and would immediately jump out (in a bad way) to any interviewer with a technical background. **Never ever ever ever modify state directly, always use setState!!**